src/assets/js/pages/EventPage/components/EventStatistics.js

```diff
@@ -44,6 +48,24 @@ export default class EventStatistics extends React.Component {
     return (
       <div className="event-statistics event-page__card">
         <h2>Applicant Demographics</h2>
+        <div className="row">
+          <h5>Gender Statistics</h5>
```

Comment:
I think it would be nice to move the Gender Statistics bit into it's own component `GenderStatistics.js` that you can import and use in [StatisticsTab.js](https://github.com/UCSDTESC/Check-in/blob/master/src/assets/js/pages/EventPage/tabs/StatisticsTab.js) and make `EventStatistics` have only a breakdown of the user statuses. I think we should also have a pie chart similar to what you have here for `GenderStatistics` for the `EventStatistics` too. (So we're splitting out the gender breakdown from this component into another and adding a pie chart to this component)